lib/stemcell/launcher.rb

```diff
@@ -181,6 +182,14 @@ def launch(opts={})
         end
       end
 
+      if opts['node_json'].is_a?(Hash)
```

Comment:
do you expect this to be a hash, or a string?

Reply:
Both, but I agree that it wasn't very clear. 

The old code would (1) turn a hash into JSON string, (2) allow a string (any string) and (3) if no string was set in step (1) or passed in, it would default to an empty JSON object.

I've now updated it to this, which is more robust:

```ruby
      if opts['node_json'].is_a?(Hash)
        opts['node_json'] = JSON.generate(opts['node_json'])
      elsif opts['node_json'].is_a?(String)
        unless valid_json?(opts['node_json'])
          raise '`node_json` must be a hash or a valid json string'
        end
      else # fallback
        opts['node_json'] = '{}'
      end
```